ingestion/src/metadata/orm_profiler/orm/converter.py

```diff
@@ -99,7 +99,9 @@ def build_orm_col(idx: int, col: Column, table_service_type) -> sqlalchemy.Colum
     """
     return sqlalchemy.Column(
         name=str(col.name.__root__),
-        type_=_TYPE_MAP.get(col.dataType),
+        type_=_TYPE_MAP.get(col.dataType)
+        if not col.arrayDataType
+        else sqlalchemy.ARRAY(item_type=col.arrayDataType),
```

Comment:
Hey @open-metadata/ingestion, what if we add an exception here to handle error, if we have that error thrown, we handle it by adding a item_type?

Reply:
this will handle other types that would fail.

Reply:
🤔 is your idea to throw a better error than what SQA does?

Reply:
so as per the issue it says that item_type reoccurs on JSON as well.. should we add try except and it will catch the attribute error, and just add item_type as part of handling that error

Reply:
never mind, 😓 i checked the sqlalchemy types and item_type only belongs to ARRAY

Reply:
so no need for error handling around here